src/main/java/stream/flarebot/flarebot/util/general/GeneralUtils.java

```diff
@@ -137,28 +133,242 @@ public static String getProgressBar(Track track) {
                 " " + GeneralUtils.percentageFormat.format(percentage) + "%";
     }
 
-    private static char getPrefix(Guild guild) {
-        return guild == null ? Constants.COMMAND_CHAR : FlareBotManager.instance().getGuild(guild.getId()).getPrefix();
+    /**
+     * Gets a String stacktrace from a {@link Throwable}
+     *
+     * @param e the {@link Throwable}.
+     * @return A string stacktrace
+     */
+    public static String getStackTrace(Throwable e) {
+        StringWriter writer = new StringWriter();
+        PrintWriter printWriter = new PrintWriter(writer);
+        e.printStackTrace(printWriter);
+        printWriter.close();
+        return writer.toString();
     }
 
-    public static String formatCommandPrefix(Guild guild, String usage) {
-        String prefix = String.valueOf(getPrefix(guild));
-        if (usage.contains("{%}"))
-            return usage.replaceAll("\\{%}", prefix);
-        return usage;
+    /**
+     * Gets an int from a String
```

Comment:
You don't mention the default value, when it returns the default value or really what the int itself is.